SpringProject/src/test/java/com/gradproject2019/UserServiceTest.java

```diff
@@ -36,12 +38,18 @@
     @Mock
     private AuthServiceImpl authServiceImpl;
 
-    private final User qwerty = new User( 1L, "qwerty", "qwerty", "qwerty", "qwerty@qwerty.com", "Qwerty!1", "qwerty", null);
-    private final User qwerty2 = new User( 2L, "qwerty2", "qwerty2", "qwerty2", "qwerty2@qwerty.com", "Qwerty!1", "qwerty2", 3L);
+    private final User user = new User(1L, "qwerty", "qwerty", "qwerty", "qwerty@qwerty.com", "Qwerty!1", "qwerty", null);
+    private final User user2 = new User(2L, "qwerty2", "qwerty2", "qwerty2", "qwerty2@qwerty.com", "Qwerty!1", "qwerty2", 3L);
+    private final User manager = new User(3L, "qwerty3", "qwerty3", "qwerty3", "qwerty3@qwerty.com", "Qwerty!3", "qwerty3", null);
+    private final User user4 = new User(4L, "qwerty4", "qwerty4", "qwerty4", "qwerty4@qwerty.com", "Qwerty!4", "qwerty4", 3L);
+    private final User user5 = new User(5L, "qwerty5", "qwerty5", "qwerty5", "qwerty5@qwerty.com", "Qwerty!5", "qwerty5", 3L);
```

Comment:
This seems quite messy to me, which is a typical sign of a code smell.
1. Do you have a builder for users? If so you should be using that instead
2. Do the other fields (besides userId and managerId) need to be here? If not then you could give them all the same default values when using the builder

Reply:
i don't need user 4 and user 5, i just wanted to check a list would be returned but i will remove them